scenarios/aws/eks/run.go

```diff
@@ -121,7 +121,8 @@ func Run(ctx *pulumi.Context) error {
 		return err
 	}
 
-	// Export the cluster's kubeconfig.
+	// Export cluster’s properties
+	ctx.Export("eks-cluster-name", cluster.EksCluster.Name())
```

Comment:
Perhaps a common `kube-cluster-name` would be easier to more work with.

Reply:
Fixed in https://github.com/DataDog/test-infra-definitions/pull/381/commits/37b7b7d9ea481f0c00c67101f89db813c23ecb98.